ompi/mca/io/romio321/romio/adio/common/ad_read_coll.c

```diff
@@ -811,32 +809,34 @@ static void ADIOI_R_Exchange_data(ADIO_File fd, void *buf, ADIOI_Flatlist_node
 
     if (buftype_is_contig) {
 	j = 0;
-	for (i=0; i < nprocs; i++) 
+	for (i=0; i < nprocs; i++) {
```

Comment:
By the Open MPI coding standards, this pair of `{}` is technically missing -- but it's not actually _needed_ here (because the body of the `for` loop is technically only 1 statement (the `if` block)).

Let's go for the minimum-distance fix here, though -- let's not add by-the-OMPI-style-it's-missing-but-it's-still-technically-correct pairs of `{}` like this one.